src/models/WorkPool.ts

```diff
@@ -11,7 +11,7 @@ export interface IWorkPool {
   isPaused: boolean,
   defaultQueueId: string,
   concurrencyLimit: number | null,
-  baseJobTemplate: Record<string, unknown>,
+  baseJobTemplate: BaseJobTemplateRequest,
```

Comment:
I'm surprised to see a request model here, shouldn't this be a mapped response model?

Reply:
Agreed, I saw this and noticed the interface didn't match the implementation. The implementation uses the request model. I went a few minutes down the road of mapping it to a new `BaseJobTemplate` model but realized I'd have to touch quite a few files to update that and ultimately decided to leave it as is for now.

Reply:
Ok since this is how the work pool model is typed already (I'm surprised that union works tbh) I see this as at least a strict improvement, if not a perfect one